src/internal/type.ts

```diff
@@ -126,3 +126,62 @@ export interface BucketItemStat {
 export type StatObjectOpts = {
   versionId?: string
 }
+
+/* Replication Config types */
+export type ReplicationRuleStatus = {
```

Comment:
maybe need to re-export these types in `minio.d.ts`?

Reply:
do we really need it to re-export . ?  could you share any downsides ?

Reply:
if you want to build a complex `ReplicationRule` with TypeScript